configuration/prod/validator_1.toml

```diff
@@ -0,0 +1,32 @@
+server_config_path = "server_1.json"
+host = "validator-1"
```

Comment:
In the current hacky implementation, the host needs to match the Workload name defined in the Kubernetes YAML files here https://github.com/linera-io/linera-infra/pull/2/files